### PR TITLE
[Feature] Card component

### PR DESCRIPTION
**Pull request by aleksik**
## Description

This PR implements the Card component.

## How Has This Been Tested?

Tested in Storybook locally.

Live demo:
https://aleksik.github.io/core/?path=/story/components-card--empty
 https://aleksik.github.io/react/?path=/docs/components-card--empty

### CODE REVIEW

**Comment by ronijaakkola** [comment]
Documentation done here: https://github.com/City-of-Helsinki/helsinki-design-system/pull/246
It should be merged into this branch and finalize it (add examples to Playground).

**Review by ronijaakkola — CHANGES_REQUESTED** [review]
image

Spacing between elements seems a bit off. I just realized Abstract had a hard time displaying those in the showcase artboard inspect. So here are those in written format:

- Spacing between the title and text should be 24px (`--spacing-m` token)
- Spacing between the text and the button should be 32px (`--spacing-l` token)

---

Before merging, we should merge documentation branch here and finalize it (add examples to Playground). Documentation is done here: #246

**Comment by aleksik** [comment]
@ronijaakkola spacing is now updated based on Abstract artboard. These fixes can be tested here:
https://aleksik.github.io/core/?path=/story/components-card--empty
https://aleksik.github.io/react/?path=/docs/components-card--empty

I also cherry-picked the documentation commits and added usage examples.

**Review by ronijaakkola — APPROVED** [review]
Hyvältä näyttää. Tein pari lisäystä dokumentaatioon ja korjasin jotain saavutettavuusongelmia (kun dokumentaatio oli ajalta ennen korjauksia).

File: packages/react/src/components/card/Card.module.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.card {
```
Comment (niglu1) [inline packages/react/src/components/card/Card.module.css:1]: Could this file be changed from `.css` to `.scss`?

File: packages/react/src/components/card/Card.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
+import React from 'react';
```
Comment (niglu1) [inline packages/react/src/components/card/Card.stories.tsx:1]: Could the stories be written using `args`?

**Review by niglu1 — CHANGES_REQUESTED** [review]
Nais 👌

**Review by niglu1 — APPROVED** [review]

**Pull request merged**